lib/p2p/Peer.ts

```diff
@@ -521,6 +521,12 @@ class Peer extends EventEmitter {
       entry.resolve(packet);
 
       this.handshakeState = packet.body;
+
+      if (this.handshakeState!.nodePubKey === this.handshakeData.nodePubKey) {
+        // TODO: remove from hosts table
```

Comment:
No it shouldn't when the pubkey is the same as yours. But this comment is obsolete because I solved it in a way that doesn't even add it to the hosts table.